docusaurus/docs/extensions/rancher-2.9-support.md

```diff
@@ -53,4 +53,4 @@ For more information about the annotation we allow for, check the documentation
 - After the above steps, just publish a new version of the extension. That published version should now be compliant with Rancher 2.9.0.
 
 
-To find out more about the support matrix for Shell versions in regards to Rancher versions, check the support matrix [here](./support-matrix#shell-support-matrix).
\ No newline at end of file
+To find out more about the support matrix for Shell versions in regards to Rancher versions, check the support matrix [here](./support-matrix#shell-support-matrix).
```

Comment:
don't see why this is highlighted as an updated 🤔 EOF?

Reply:
probably.... 😛  don't worry